### DISCUSSION TITLE
monai_trainer.py and ignite API

### DISCUSSION THREAD

**Discussion by vfdev-5** [NVIDIA/NVFlare#865] [Q&A] [answered]
https://github.com/NVIDIA/NVFlare/blob/88c94191c7edfaf151142c347aa5ec87a9e3c517/examples/hello-monai/app/custom/monai_trainer.py#L174

I was checking project's examples and have seen above comment. I wonder whether it is usage question or ignite is still missing a feature ? Thanks

**Comment by YuanTingHsieh** [comment, marked as answer]
Thanks for raising the discussion.

@yiheng-wang-nv did you write this line at that time? do you remember why you make this comment?

**Reply by holgerroth** [reply]
Thanks, @vfdev-5. This example will be replaced by [one](https://github.com/NVIDIA/NVFlare/tree/2.2/integration/monai/examples/spleen_ct_segmentation) using Ignite's new API (available in 0.4.10) and MONAI's new FL components, i.e. [MonaiAlgo](https://github.com/Project-MONAI/MONAI/blob/dev/monai/fl/client/monai_algo.py).

**Reply by YuanTingHsieh** [reply]
@holgerroth does that mean we need to remove hello-monai here?

**Reply by holgerroth** [reply]
Yes, I think so. We already replaced the `hello-monai-bundle` in 2.2. https://github.com/NVIDIA/NVFlare/tree/2.2/examples/hello-monai-bundle but not in the dev branch. I will make the same change there and remove `hello-monai`

**Reply by holgerroth** [reply]
See https://github.com/NVIDIA/NVFlare/pull/938
